Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update && apt-get install -y \
     apt-get -y autoremove && \
     apt-get -y clean  && \
     rm -rf /var/lib/apt/lists/*
+WORKDIR /code
 RUN pear install PHP_CodeSniffer
 RUN ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
Sounds good. Making it as early as it can be and grouped with other things related to it makes sense.